Extending/Key-Vault/index.md

```diff
@@ -62,6 +67,117 @@ There are different ways to access the Azure Key Vault. It is important that the
+
+![image](https://user-images.githubusercontent.com/11179749/196052668-124d1496-4486-4098-9198-eff809876c80.png)
+
+Enter the GUID we took note of earlier into the **Search Box**, you should see your Web App listed. 
+
+![image](https://user-images.githubusercontent.com/11179749/196052706-15431bf4-80ea-4bb7-b40e-ebda45264fb7.png)
+
+Click your Web App to Select and click Next and then Create:
+
+![image](https://user-images.githubusercontent.com/11179749/196052849-970a97c5-e945-415a-9469-a67f485424ea.png)
+
+If you visit the **Access Policies** section again you should now see your web app in the list and its permissions:
+
+![image](https://user-images.githubusercontent.com/11179749/196052924-0d0559c0-a414-4bbd-ab91-94fc25dc720f.png)
+
+### Link our Key Vault Secret to an Azure Web App
+
+In your Azure Web App head to **Configuration** under **Settings**
+
+![image](https://user-images.githubusercontent.com/11179749/196053006-3a95fc5f-1038-4228-9ae4-467050ea5759.png)
+
+Here we can add App Settings and Connection Strings to the environment. Lets start off with the Umbraco Database Connection String.
+
+Under Connection Strings, click **Advanced Edit**
+
+![image](https://user-images.githubusercontent.com/11179749/196053130-8fb6c2b9-61c7-4c02-a419-8570174c6646.png)
+
+You can obtain the Secret Uri by visitng the specific version of your secret and copying the Url:
+
+![image](https://user-images.githubusercontent.com/11179749/196054001-cc215c04-d29c-435a-ae7b-6e8efb7f3faa.png)
+
+Then paste in the following JSON Object inside the square brackets. Ensure you update `{keyvault-name}`, `{secret-name}` and `{version-id}`.
+```json
+{
+    "name": "umbracoDbDSN",
+    "value": "@Microsoft.KeyVault(SecretUri=https://{keyvault-name}.vault.azure.net/secrets/{secret-name}/{version-id}/)",
+    "type": "Custom",
+    "slotSetting": false
+}
+```
+The ID is optional but recommended as it enables you to control which version of the secret is used at your discretion. Leave it out if you always want the Web App to pull the latest version of the secret.
+
+Wait a moment and refresh the screen, you should see a Green tick. If you don't have a Green tick you need to review your Access Policies in the previous step.
+
+![image](https://user-images.githubusercontent.com/11179749/196053419-f53feba2-b8ed-4b98-99f0-ee68f58ac8e4.png)
+
+
+We will perform the same approach for our App Settings. We will be updating the following App Settings for Azure Blob Storage.
+
+```json
+"Umbraco": {
+    "Storage": {
+      "AzureBlob": {
+        "Media": {
+          "ConnectionString": "",
+          "ContainerName": ""
+        }
+      }
+    }
+```
+
+Due to the secrets being nested we need to use double underscore `__` to correctly reference the value on our Web App.
+
+On the Web App click **Advanced Edit** for Application Settings:
```

Comment:
```suggestion
On the Web App select **Advanced Edit** for Application Settings:
```

Reply:
I think its important to highlight that names need double underscore in its name to reference nested JSON properties in Azure. Unless you think the example is clear enough. I like to reiterate 😄

Reply:
Oh! I'm not suggestion to remove anything, simply changing "click" to "select".
Not sure why it's highlighting some of the text in read here 🤔